auth/auth0/README.md

```diff
@@ -0,0 +1,90 @@
+# auth0-authentication
+
+Add Auth0 Authentication to your Graphcool project 🎁
+
+
+## Getting Started
+
+### 1. Add the template to your Graphcool service
+
+```sh
+graphcool add-template graphcool/templates/auth/auth0
+```
+
+### 2. Uncomment lines in `graphcool.yml` and `types.graphql`
+
+The [`add-template`](https://docs-next.graph.cool/reference/graphcool-cli/commands-aiteerae6l#graphcool-add-template) command is performing three major steps:
+
+1. Download the source files from the [`src`](./src) directory and put them into your service's `src` directory (into a subdirectory called `auth0`).
+2. Download the contents from [`graphcool.yml`](./graphcool.yml) and append them as comments to your service's `graphcool.yml`.
+3. Download the contents from [`types.graphql`](./types.graphql) and append them as comments to your service's `types.graphql`.
+
+In order for the changes to take effect, you need to manually uncomment all the lines that have been added by the `add-template` command.
+
+### 3. Setup Auth0
+
+#### 3.1 Create an API
+
+* Create a new Auth0 account or log-in to your existing one
+* Create a new API. Select `RS256` as the signing algorithm. Keep the Identifier value handy as we will use it later on.
+
+![](./docs/create-api.png)
+
+#### 3.2 Create a Client
+
+* Create a new Auth0 client with the following settings
+
+![](./docs/new-client.png)
+
+  * Client Type - *Regular Web Application*
```

Comment:
Why "Regular Web Application" and not "Single Page App"? If you're using the lock widget I assume you're authenticating via client-side JavaScript, no?

Reply:
It can also be "Native" if you are in react-native :wink:

Reply:
I choose "Regular Web Application" because it seemed to be the most generic one, but you raise a good point. @marktani any thoughts ?